app/javascript/packs/base.js

```diff
@@ -4,6 +4,9 @@ import {
   getInstantClick,
   initializeTouchDevice,
 } from '../topNavigation/utilities';
+import { getFocusTrapToggle } from '../utilities/getFocusTrapToggle';
```

Comment:
refactor (blocking): Use the webpack @utilities alias.

```suggestion
import { getFocusTrapToggle } from '@utilities/getFocusTrapToggle';
```